authors/apps/authentication/tests/test_user_registration.py

```diff
@@ -129,9 +130,52 @@ def test_user_log_in_credentials_miss_password_and_email(self):
         self.assertIn(response.data["errors"]["email"][0], 'This field is required.')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_get_current_user_jwt(self):
+        """
+        Get current user from JWT
+        """
+        self.client.post('/api/users/register/', login_credentials_data, format='json')
```

Comment:
@dannylwe the repeated process of generating the token should be done in the test base case.